fcli-core/fcli-common/src/main/resources/com/fortify/cli/common/i18n/FortifyCLIMessages.properties

```diff
@@ -18,21 +18,16 @@ usage.parameterListHeading = %nCommand parameters%n
 usage.optionListHeading = %nCommand options%n
 
 # Generic, non command-specific options
-fcli.genericOptions.heading = Generic fcli options%n
-help = Show this help message and exit. Use 'fcli <command> -h' to display help for subcommands.
+fcli.genericOptions.heading = Generic fcli options (see documentation link below)%n
```

Comment:
Please change to `Generic fcli options (also see documentation link below)%n` (add 'also')